extensions/skins/conversation_v1/Conversation.js

```diff
@@ -647,13 +647,42 @@ oppia.directive('answerFeedbackPair', [function() {
   return {
     restrict: 'E',
     scope: {
-      answer: '&',
-      feedback: '&',
+      data: '=',
       oppiaAvatarImageUrl: '&',
       profilePicture: '&',
-      shortAnswer: '&'
+      currentProgressDotIndex: '&'
```

Comment:
It's really the index of the active (i.e., currently-showing) card. I was thinking of renaming it to activeCardIndex, but then thought I might save that for a later pass.